src/BTDeviceModel.cpp

```diff
@@ -30,13 +31,26 @@ BTDeviceModel::BTDeviceModel(QObject* parent)
     : QAbstractListModel(parent)
     , d(new Private)
 {
+    readDeviceNames();
 }
 
 BTDeviceModel::~BTDeviceModel()
 {
     delete d;
 }
 
+void BTDeviceModel::readDeviceNames()
+{
+    deviceNames.clear();
```

Comment:
It seems to me like we'd probably want to change to asking the Settings class for this information... Don't worry about doing that in this patch, but if you could pop a /// TODO in there to suggest that bit of future work, that'd be lovely, thank you :)